assets/src/design-system/components/dropDown/menu/defaultListItem.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { forwardRef } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Checkmark } from '../../../icons';
+import { THEME_CONSTANTS } from '../../../theme';
+import { DROP_DOWN_ITEM } from '../types';
+import { ListItem, ListItemDisplayText } from './components';
+
+const DefaultListItem = ({ option, isSelected, ...rest }, ref) => (
+  <ListItem
+    {...rest}
+    ref={ref}
+    isSelected={isSelected}
+    disabled={option.disabled}
+  >
+    {isSelected && (
+      <Checkmark
+        data-testid={'dropdownMenuItem_active_icon'}
+        aria-label={__('Selected', 'web-stories')}
+      />
+    )}
+    <ListItemDisplayText
+      as="span"
+      size={THEME_CONSTANTS.TYPOGRAPHY.TEXT_SIZES.SMALL}
+    >
+      {option?.label}
```

Comment:
Option isn't used as `null` above on `41` would it be here?

Reply:
good point, removed it! There's no need for it.